lib/qesapdeployment.pm

```diff
@@ -337,8 +337,9 @@ sub qesap_sh_destroy {
 
 =head3 qesap_execute
 
-    qesap_execute(cmd => $qesap_script_cmd [, verbose => 1] );
+    qesap_execute(cmd => $qesap_script_cmd [, verbose => 1, cmd_options => $cmd_options] );
 
+    cmd_options - allows to append additional qesap.py commans arguments like "qesap.py terraform -d"
```

Comment:
Does could be the explanation and the example like

> ... like : 
```
qesap_execute(cmd => 'terraform', cmd => '-d')
```
result in 
```
qesap.py terraform -d
```

Reply:
Yep. That is a good idea. 
Fixed and pushed.